pkg/flare/archive.go

```diff
@@ -544,6 +550,56 @@ func writeConfigCheck(tempDir, hostname string, data []byte) error {
 	return err
 }
 
+// Used for testing mock HTTP server
+var taggerListURL string
+
+func zipTaggerList(tempDir, hostname string) error {
+	f := filepath.Join(tempDir, hostname, "tagger-list.log")
+	err := ensureParentDirsExist(f)
+	if err != nil {
+		return err
+	}
+
+	w, err := newRedactingWriter(f, os.ModePerm, true)
+	if err != nil {
+		return err
+	}
+	defer w.Close()
+
+	c := api_util.GetClient(false) // FIX: get certificates right then make this true
+
+	err = api_util.SetAuthToken()
```

Comment:
Perhaps I'm missing something, if the agent is up, the token should already be set. If the agent is not up and we try to generate a flare the `taggerListURL` will expectedly time out. I am not entirely sure why we'd want to set an auth token here - the client returned by `api_util.GetClient(false)` will already add the bearer token to the requests so I think that's enough?

Reply:
you are right; i was doing most of my testing using the `local` flag (`agent flare -l`) so that's why i added this. i was confused by the help message that suggests using the flag in containerized environments.

thanks for bringing this up!!